demo/draw-path/js/style.js

```diff
@@ -0,0 +1,58 @@
+class Style {
+
+    #bpmnElementsRegistry;
+
+    constructor(bpmnElementsRegistry) {
+        this.#bpmnElementsRegistry = bpmnElementsRegistry;
+    }
+
+    /**
+     * @param ids can be an array or a string
+     * @private
```

Comment:
**nitpick:** the method is public, so I guess the `private` annotation should be removed from the JSDoc.
Same remark for all methods of this class where it applies.

Reply:
Good catch :+1: